CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 * [ENHANCEMENT] Query-frontend / Querier: increase internal backoff period used to retry connections to query-frontend / query-scheduler. #3011
 * [ENHANCEMENT] Querier: do not log "error processing requests from scheduler" when the query-scheduler is shutting down. #3012
 * [ENHANCEMENT] Query-frontend: query sharding process is now time-bounded and it is cancelled if the request is aborted. #3028
+* [ENHANCEMENT] mimirtool analyze: Store the query errors instead of exit during the analysis. [cortex-tools: #236](https://github.com/grafana/cortex-tools/issues/236)
```

Comment:
A couple of requests please:
1. Could you move it below, under `### Mimirtool`?
2. Instead of linking `[cortex-tools: #236](https://github.com/grafana/cortex-tools/issues/236)`, please just reference this PR number `#3052`

Reply:
Thanks, done!